test/multi_rollup_sequencer_test.go

```diff
@@ -0,0 +1,151 @@
+package test
+
+import (
+	"context"
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/rollkit/go-sequencing"
+)
+
+func TestMultiRollupSequencer_SubmitRollupTransaction(t *testing.T) {
+	sequencer := NewMultiRollupSequencer()
+
+	rollupId := []byte("test-rollup")
+	tx := []byte("transaction data")
```

Comment:
I think it would be good to add more randomness to our testing. These are easy edits to help add more randomness.
```suggestion
	rollupId := randBytes()
	tx := randBytes()
	...
func randBytes() []byte {
        len := rand.Intn(20)
        buf = make([]byte, len)
        _, err := rand.Read(buf)
        if err != nil {
                panic(err)
        }
        return buf
}
```